utils/token.ts

```diff
@@ -1,2 +1,5 @@
 export const shortenAddress = (address: string) =>
   `${address.slice(0, 6)}...${address.slice(-4)}`;
+
+export const gnosisExplorerAddress = (address: string) =>
```

Comment:
Minor, non-blocking:
This naming feels weird to me, it's a function but I'd expect maybe a string from this variable name.

WDYT about renaming it into a verb, like `exploreGnosisAddress` or something like that?

Reply:
I'll take it.

Reply:
I would call it `getGnosisAddressExplorerLink` instead